service/collector/issue.go

```diff
@@ -0,0 +1,157 @@
+package collector
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+	"github.com/google/go-github/github"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+var (
+	issueLabelsDesc *prometheus.Desc = prometheus.NewDesc(
+		prometheus.BuildFQName(namespace, subsystem, "labels_count"),
+		"Github issue labels.",
+		[]string{
+			labelOrg,
+			labelRepo,
+			labelLabel,
+			labelState,
+		},
+		nil,
+	)
+	issueStatesDesc *prometheus.Desc = prometheus.NewDesc(
+		prometheus.BuildFQName(namespace, subsystem, "states_count"),
+		"Github issue states.",
+		[]string{
+			labelOrg,
+			labelRepo,
+			labelState,
+		},
+		nil,
+	)
```

Comment:
For now there are only two metrics supported. We can extend this and go from here.